_docs/howto/host-from-home.md

```diff
@@ -0,0 +1,27 @@
+---
+title:    Host from Home
+category: Get Started
+order:    4
+---
+
+Any site you [create](/docs/howto/create-a-site.html) or save will be hosted from your computer. To save: click the <span class="fa fa-floppy-o"></span> in the URL bar.
+
+<figure>
+  <img class="bordered" src="/img/screenshot-save-btn.png">
+  <figcaption>Save a site to host it from your computer.</figcaption>
+</figure>
+
+
+The [Dat Protocol](/docs/dat/intro.html) encrypts network traffic so that only users who possess the URL of a site can read its content. If you only share the URL with your intended recipients, the site files and metadata will be secret.
```

Comment:
This is an important feature, and I think we should highlight that. Let's consider adding a `<h3>` with the title "Sharing your site" or something that indicates it's an important point. Other suggestions?